tests/HtmlConverterTest.php

```diff
@@ -22,6 +22,7 @@ public function test_empty_input()
     public function test_plain_text()
     {
         $this->html_gives_markdown('test', 'test');
+        $this->html_gives_markdown('test1&test2&amp;test3&amp;amp;test4&amp;amp;amp;test5', 'test1&test2&test3&amp;test4&amp;amp;test5');
```

Comment:
Would you mind also adding the test cases from https://github.com/thephpleague/html-to-markdown/issues/88#issuecomment-179285003 (first example) and https://github.com/thephpleague/html-to-markdown/issues/60#issue-111121197?  I want to ensure both of these will result in the correct conversions.